### DISCUSSION TITLE
Fetch and store link preview for url field

### DISCUSSION THREAD

**Discussion by mahirchavda** [pocketbase/pocketbase#1666] [Q&A] [answered]
I know pocketbase has the url field. so i was wondering if you can provide on option to fetch and store the link preview related data automatically (like thumb, type, title, description).

**Comment by ganigeorgiev** [comment, marked as answer]
I'm not sure that I understand what you are asking.

The `url` field is just a string with url validations. There is no integrated "scraping"-like behavior, if that's what you are asking.

**Reply by mahirchavda** [reply]
Right. I wanted to check if that can be added as a feature in the future?

**Reply by ganigeorgiev** [reply]
No, sorry. There are no plans for that.
If you haven't already, you can check the current roadmap and tasks priority in https://github.com/orgs/pocketbase/projects/2.
